Annotato/Annotato/ViewModels/Document/DocumentAnnotationViewModel.swift

```diff
@@ -29,4 +46,31 @@ class DocumentAnnotationViewModel {
     var startingAnnotationType: AnnotationType? {
         isNew ? self.parts[0].annotationType : nil
     }
+
+    func updateLocation(documentViewPoint: CGPoint, parentView: UIView?) {
+        guard let parentView = parentView else {
+            return
+        }
+        guard let pdfView = parentView.superview?.superview
```

Comment:
Seems a bit dangerous to call `.superview.superview` as we never know if the parent view could be moved and nested in another layer in future. This will cause the function to fail silently. Refer to next comment for suggested fix.

Reply:
I replied below